### DISCUSSION TITLE
please allow disabling "for you" feed

### DISCUSSION THREAD

**Discussion by twojstaryzdomu** [community/community#15915] [Other Feature Feedback, Questions, & Ideas]
Please allow disabling "for you" feed. This is not a social network. I simply do not wish to see some random content when accessing github.com.

**Comment by DennisSmuda** [comment]
I just got a "Recommended for you" (don't want to bash any specific repo, so not gonna name) but I got a little excited, only to find out that it's basically a *stale repo* that hasn't been updated in 4-5 years.

That was actually a pretty huge disappointment, and an emotion that I don't wanna feel on this site - so I have to agree with your sentiment @twojstaryzdomu. 

I guess it's supposed to be a "discovery-feature", but I feel like that problem is already solved. There is already a big "Explore"-section (which I very much like!). And there are "awesome-lists" for pretty much any topic.

So I'm asking myself:
Is github trying to show me *specific* stuff that I didn't find on my own before? (didn't work so far)
or is github trying to "broaden" my horizon and try to show me things that I don't know were interesting to me?

Both don't seem like a problem that programmers are really having on a platform they use for version control.

**Reply by twojstaryzdomu** [reply]
I ask myself another one: how inept does github think the users of this site are in finding stuff that interests them? I mean people come here with a specific focus or they may simply use the search facility.

Why risk the ire of developers, many of whom will be privacy-oriented by trade, and triggered at the very mention of algorithmic use of their behavioural data?

On day one the algorithmic feed went live, it should have been opt-in. Right now, it's still not that.

I doubt there is any great need for developers to follow each other's commits, unless they are already involved in a common project, the case which makes the algorithmic fee redundant.

**Reply by smysmymrik1803** [reply]
Super

**Reply by AndASM** [reply]
> I ask myself another one: how inept does github think the users of this site are in finding stuff that interests them? I mean people come here with a specific focus or they may simply use the search facility.

Well if they limit search and filter it through an "AI" algorithm, then obviously the users won't be able to find the stuff that interests them. Thus creating demand for whatever content GitHub deems appropriate... right? Only a bit of a conflict of interest here, there, and everywhere.

**Comment by jareks** [comment]
I wholeheartedly support this thread - I *do not* want any pointless distractions on github.

**Comment by otech47** [comment]
please god no more algorithmic feeds

**Comment by danielniccoli** [comment]
I don't like this feed. I'm much more interested what's happening in repositories I contribute to – be it code, or issues.  
The feed is **extremely** distracting and it's so confusing, it makes me forget what I came to GitHub for for a couple of seconds.

**Comment by siminapasat** [comment]
Hi folks! thank you for your feedback. We're iterating on the feed, and also on the general github.com dashboard for signed in users, to hopefully make it more useful.

**Reply by danielniccoli** [reply]
> to hopefully make it more useful

This thread is not about "making it more useful", it's about letting us disable it. Please don't twist this discussion into something, only because you don't want to hear that the people here don't want it at all in any form or shape.

**Comment by NellyisDevv** [comment]
I've been thinking this for a really long time, It would be really nice to just have the option to open GitHub and just have no distractions just a home page, this should definitely be an option, although I don't hate the feeds.

**Reply by jamieshark** [reply]
> just a home page

What would a distraction-free home page look like for you?

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by AndASM** [reply]
It still defaults to the "For you" tab. Forcing users into a "value add" feature. A gross thing for GitHub to do. What, did they get bought out by some giant corporation that is notorious for unethical behavior or something?

**Comment by scott-rothman** [comment]
Make the feed go away.  I come to github to look at my code not for a social media site.
